src/main/java/spoon/reflect/reference/CtExecutableReference.java

```diff
@@ -55,9 +55,18 @@
 	 */
 	Constructor<?> getActualConstructor();
 
+	@Override
 	CtExecutable<T> getDeclaration();
 
 	/**
+	 * Returns a subtype {@link CtExecutable} that corresponds to the reference
+	 * even if its declaring type isn't in the Spoon source path.
```

Comment:
in the Spoon source path (in this case, the Spoon elements are built with runtime reflection)